plugin/sampling/strategyprovider/adaptive/options.go

```diff
@@ -40,34 +40,30 @@ const (
 // The abbreviation SPS refers to "samples-per-second", which is the target
 // of the optimization/control implemented by the adaptive sampling.
 type Options struct {
-	// TargetSamplesPerSecond is the global target rate of samples per operation.
-	// TODO implement manual overrides per service/operation.
-	TargetSamplesPerSecond float64 `mapstructure:"target_samples_per_second"`
+	Calculation Calculation `mapstructure:"calculation"`
+	Sampling    Sampling    `mapstructure:"sampling"`
```

Comment:
I'm not sure this is an improvement. Most flags are related to calculation anyway. Some are related to boundary conditions, like min rate.